aptos-move/framework/supra-framework/sources/pbo_delegation_pool.move

```diff
@@ -1455,8 +1455,9 @@ module supra_framework::pbo_delegation_pool {
             / unlock_schedule.period_duration;
         let last_unlocked_period = unlock_schedule.last_unlock_period;
         let schedule_length = vector::length(&unlock_schedule.schedule);
+        let one = fixed_point64::create_from_rational(1,1);
         let cfraction = unlock_schedule.cumulative_unlocked_fraction;
```

Comment:
Please also add an early return at the beginning of the function if this value is already `1` when it is invoked.

Reply:
Done.